central/centralhealth/service/service_impl.go

```diff
@@ -46,6 +49,38 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 
 // GetUpgradeStatus returns the upgrade status for Central.
 func (s *serviceImpl) GetUpgradeStatus(ctx context.Context, empty *v1.Empty) (*v1.GetUpgradeStatusResponse, error) {
+	// TODO: ROX-12059 Need to figure out if there is anything we can do with size in Postgres as that data
+	// IS NOT accessible from central
+	if features.PostgresDatastore.Enabled() {
+		upgradeStatus := &v1.CentralUpgradeStatus{
+			Version: version.GetMainVersion(),
+			//CanRollbackAfterUpgrade:               int64(freeBytes)+toBeFreedBytes > requiredBytes,
+			//SpaceAvailableForRollbackAfterUpgrade: int64(freeBytes) + toBeFreedBytes,
+			//SpaceRequiredForRollbackAfterUpgrade:  requiredBytes,
+		}
+
+		// Get a short-lived connection for the purposes of checking the version of the replica.
+		_, adminConfig, err := pgconfig.GetPostgresConfig()
+		if err != nil {
+			return nil, err
+		}
+		pool := pgadmin.GetReplicaPool(adminConfig, "central_previous")
+		defer pool.Close()
+
+		// Get rollback to version
+		migVer, err := migrations.ReadVersion(pool)
+		if err != nil {
+			log.Infof("Unable to get previous version, leaving ForceRollbackTo empty")
```

Comment:
log the error?

Reply:
@connorgorman yeah, if you look below we never returned that particular error.  we would simply set the rollbackTo version if the err was nil and the version was populated.  So I logged it in the interest of keeping it functionally the same as before.  We can easily return it if you would rather.